android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt

```diff
@@ -538,7 +539,9 @@ constructor(
         // the observations would have key as coding.code=populationId and value as codeableConcept
         observations
           .groupBy { it.codingOf(POPULATION_OBS_URL)?.display }
+          .asSequence()
           .filter { it.key.isNullOrBlank().not() }
+          .toList()
```

Comment:
Is this toList() necessary? Sequence is not performant when you collect the result